### ISSUE TITLE
Can't join a table with multiple relations and a unauthorized column

### ISSUE THREAD

**Issue by sebj54** [mevdschee/php-crud-api#727]
Hi there!

This problem is hard to explain with a few words so I'll give you an example directly:

I have two tables: `post` and `user`. A post has two relations with the user table: `author` and `moderator`. In my app, everyone can see who is the author of a given post but only admins and moderators can see the user who has moderated the post. 

So I'm using the `authorization` middleware to allow access to the `moderator` column, based on user's role (I do that with `authorization.columnHandler`).

Then, when I'm trying to list posts with a GET on `/records/post?join=user`, this error occurs:

```json
{
  "code":9999,
  "message":"Call to a member function getName() on null"
}
```

```
X-Exception-File
	[...]\vendor\mevdschee\php-crud-api\src\Tqdev\PhpCrudApi\Record\RelationJoiner.php:150
X-Exception-Message
	Call to a member function getName() on null
X-Exception-Name
	Error
```

I have a simple fix for that, see related PR. 

⚠ I really don't know if my fix does not have any side effect.

**Comment by mevdschee** [comment]
Thank you, I will review your PR shortly

**Comment by mevdschee** [comment]
I have reviewed the PR and it is merged. Thank you for your contribution. It will be released shortly.

**Comment by mevdschee** [comment]
A new version is released: v2.9.25 Bugfix null relations

**Comment by sebj54** [comment]
You're the one to be thanked! :)
